mapeditor/inspector/inspector.cpp

```diff
@@ -711,6 +720,13 @@ void Inspector::setProperty(CGHeroInstance * o, const QString & key, const QVari
 		o->randomizeArmy(o->getHeroType()->heroClass->faction);
 		updateProperties(); //updating other properties after change
 	}
+
+	if(key == "Patrol radius")
+	{
+		auto radius = value.toInt();
+		o->patrol.patrolRadius = radius;
+		o->patrol.patrolling = radius != std::numeric_limits<ui32>::max();
```

Comment:
this doesn't look nice of course.

if mingw is giving a hard time for w/e reason, simply define the value of the static variable in the cpp file:
```cpp
// h
static constexpr ui32 NO_PATROLLING; // maybe you'll have to use const instead of constexpr

// cpp
CGHeroInstance::NO_PATROLLING = std::numeric_limits<ui32>::max();
```